3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -344,6 +322,28 @@ def initialize(debug, staging, enrich, staging_stream_enrich, shred, es, archive
           @jobflow.set_task_instance_group(instance_group)
         end
 
+        # EBS
+        unless config[:aws][:emr][:jobflow][:core_instance_ebs].nil?
+          ebs_bdc = Elasticity::EbsBlockDeviceConfig.new
+
+          ebs_bdc.volume_type          = config[:aws][:emr][:jobflow][:core_instance_ebs][:volume_type]
+          ebs_bdc.size_in_gb           = config[:aws][:emr][:jobflow][:core_instance_ebs][:volume_size]
+          ebs_bdc.volumes_per_instance = 1
+          if config[:aws][:emr][:jobflow][:core_instance_ebs][:volume_type] == "io1"
+            ebs_bdc.iops = config[:aws][:emr][:jobflow][:core_instance_ebs][:volume_iops]
+          end
+
+          ebs_c = Elasticity::EbsConfiguration.new
+          ebs_c.add_ebs_block_device_config(ebs_bdc)
+          ebs_c.ebs_optimized = true
+
+          unless config[:aws][:emr][:jobflow][:core_instance_ebs][:ebs_optimized].nil?
+            ebs_c.ebs_optimized = config[:aws][:emr][:jobflow][:core_instance_ebs][:ebs_optimized]
+          end
+
+          @jobflow.set_core_ebs_configuration(ebs_c)
+        end
```

Comment:
You moved EBS here to solve [this problem](https://github.com/snowplow/snowplow/issues/4285#issuecomment-590319351) right?

Reply:
Yes. Moving the code to a different location was all that was required.